server.js

```diff
@@ -132,7 +136,13 @@ class StaticmanAPI {
 
   requireService (services) {
```

Comment:
We may want to consider adding another parameter like 

```suggestion
  requireService (services, optional = false) {
```

We probably don't want to allow null values for most situations, just specifically for backwards compatibility on the webhook endpoint. Obviously we'd need to modify the implementation of this function to match

Reply:
Changed as requested.